gitlab/unit-tests.sh

```diff
@@ -17,4 +18,9 @@ cd /opt/domjudge/domserver
 export APP_ENV="test"
 
 # Run phpunit tests.
-lib/vendor/bin/phpunit -c webapp/phpunit.xml.dist --log-junit ${CI_PROJECT_DIR}/unit-tests.xml --colors=never
+apt update
```

Comment:
Although we build this in the docker image later I assume, why do we update twice?

Reply:
Probably copy/paste error.